101-vm-simple-rhel/azuredeploy.json

```diff
@@ -24,25 +24,31 @@
       "metadata": {
         "description": "The RHEL version for the VM. This will pick the latest available image of this given RHEL version."
       }
+    },
+    "vmName": {
+      "type": "string",
```

Comment:
Is a "vmName" parameter really necessary? It should be fine to just use "[concat('rhelVM', uniqueString(resourceGroup().id))]" or something similar to that as a variable.